### DISCUSSION TITLE
Uniswap Token Approval

### DISCUSSION THREAD

**Discussion by KrautMc** [Uniswap/interface#6290] [General]
Hello, 
I've been trying to get a token approved on Uniswap but it just keeps bringing me back to approve the token again. I've tried two different browsers(Brave, Microsoft Edge), I've cleared cache, updated all apps, increased gas fee, enabled blind signing on my Ledger. I've tried everything.

**Comment by Vladadaa** [comment]
Hello

**Comment by JaffaSweet** [comment]
good idea

**Comment by ProtsVol** [comment]
Unfortunately

**Comment by tanushshshka2** [comment]
great

**Comment by simonbailey1** [comment]
Thanks, i hope i can do it too

**Comment by vladddimirb** [comment]
Good!

**Comment by docdim506** [comment]
!!

**Comment by kalinkavita** [comment]
It happened, you need to try later

**Comment by Chinzas** [comment]
im really like it

**Comment by vsi-1975** [comment]
Good

**Comment by BritvaBo** [comment]
so popular problem

**Comment by N-earning** [comment]
Hope it's solved?

**Comment by alinabdusat** [comment]
It is great man

**Comment by arturbebov45** [comment]
This is incredible

**Comment by richardgruz11** [comment]
It was helpful. Thank you!

**Comment by stanislavogureckij** [comment]
Need to change wallet

**Comment by WhiteZip57** [comment]
Bad luck. And what way did you find?

**Comment by MolodoyLB** [comment]
it may mean that you don't need to trade this token)

**Comment by binanceotsutki** [comment]
Great, very useful, thanks

**Comment by Olenazel** [comment]
super

**Comment by Shegy1994** [comment]
good!

**Comment by dianabogd** [comment]
cool, thank you

**Reply by zvezdabori4** [reply]
is a great idea!!

**Comment by AgularJU** [comment]
Hi all. I had the same story, an error came out, I thought that the metamask was buggy.

**Comment by Wallker7** [comment]
Maybe we need to build a chain there manually?

**Comment by JanetSS** [comment]
I hope I don't encounter such a problem.

**Comment by FabianHORNN** [comment]
maybe you just need to reinstall the wallet? Lately, something has been wrong with the wallet, and you’ll throw it away.

**Comment by evgreen85** [comment]
the same issue

**Comment by Crypton-56** [comment]
Why you ask many questions ?

**Comment by Angel87578** [comment]
Great, thank you

**Comment by nichechristie** [comment]
> Hello, 
> 
> I've been trying to get a token approved on Uniswap but it just keeps bringing me back to approve the token again. I've tried two different browsers(Brave, Microsoft Edge), I've cleared cache, updated all apps, increased gas fee, enabled blind signing on my Ledger. I've tried everything.

**Comment by Nika250** [comment]
good idea, thanks!

**Comment by Vlada3333** [comment]
great, thanks!

**Comment by nichechristie** [comment]
What’s the good idea 


Sent from Yahoo Mail for iPhone


On Thursday, January 23, 2025, 11:10 PM, Vlad ***@***.***> wrote:




great, thanks!
